app.js

```diff
@@ -1,12 +1,37 @@
 const express = require("express");
 const app = express();
 app.use(express.json());
-const { getTopics } = require("./controllers/topic-controller");
+const {
+  getTopics,
+  getArticles,
+  getArticleById,
+} = require("./controllers/topic-controller");
 
 app.get("/api/topics", getTopics);
 
-app.use((request, response, next) => {
+app.get("/api/articles", getArticles);
+
+app.get("/api/articles/:article_id", getArticleById);
+
+app.use((err, request, response, next) => {
```

Comment:
If you wanted to you could refactor your errors into their own directory. I would make the app file much neater.